src/app/client/reducers/register.js

```diff
@@ -0,0 +1,18 @@
+import Immutable from 'immutable';
+
+import { ADD_USER } from '../actions'
+
+
+
+export function register(state = Immutable.fromJS({}),action){
+	switch(action.type){
+		case ADD_USER:
+			state = Immutable.fromJS(action.payload)
+			console.log(state)
+			return _.clone(state)
```

Comment:
this is not required. immutablejs already done clone internally its not needed again